### PR TITLE
fix significant linting issues, remove 12 and iojs and add 4 other mi…

### PR DESCRIPTION
**Pull request by lloydbenson**
…nor fixes

### CODE REVIEW

**Comment by lloydbenson** [comment]
@ulrikaugustsson any updates on this PR?

File: lib/index.js [hunk 1]
```diff
@@ -1,103 +1,103 @@
...
 
 var Dgram = require('dgram');
-var Os = require('os');
-var Url = require('url');
 var GroupBy = require('lodash.groupby');
 var Hoek = require('hoek');
+var Os = require('os');
 var Stringify = require('json-stringify-safe');
 var Squeeze = require('good-squeeze').Squeeze;
+var Url = require('url');
```
Comment (ulrikaugustsson) [inline lib/index.js]: Shouldn't the modules that comes from node be first and then modules from npm?

**Comment by lloydbenson** [comment]
I thought it was just alphabetical but I changed it back.

**Comment by ulrikaugustsson** [comment]
Thanks!

**Pull request merged**